spec/spec_helper.rb

```diff
@@ -136,7 +136,7 @@ def fixture_pod_target_with_specs(specs, host_requires_frameworks = false, user_
 
 def fixture_aggregate_target(pod_targets = [], target_definition = nil)
   target_definition ||= pod_targets.flat_map(&:target_definitions).first || fixture_target_definition
-  Pod::AggregateTarget.new(config.sandbox, false, {}, [], target_definition, config.sandbox.root.dirname, nil, nil, pod_targets)
+  Pod::AggregateTarget.new(config.sandbox, false, { 'Release' => :release }, [], target_definition, config.sandbox.root.dirname, nil, nil, pod_targets)
```

Comment:
would be nice to extract a default constant within `target.rb` `DEFAULT_ARCHS = { 'Release' => :release, 'Debug' => :debug)`